tests/helics/test_files/example_value_fed_bad.toml

```diff
@@ -0,0 +1,24 @@
+#this is a toml configuration file for HELICS value federate
+#example toml configuration file for a value federate all arguments are optional
```

Comment:
For the "bad" example files it would be good to have a comment at the top so people using them for reference know what the issue to avoid is.